tests/TestCase/Mailer/EmailTest.php

```diff
@@ -100,14 +100,24 @@ class EmailTest extends TestCase
     public $fixtures = ['core.users'];
 
     /**
+     * @var \Cake\Mailer\Email
+     */
+    protected $Email;
+
+    /**
+     * @var array
+     */
+    protected $transports = [];
+
+    /**
      * setUp
      *
      * @return void
      */
     public function setUp()
     {
         parent::setUp();
-        $this->CakeEmail = new TestEmail();
+        $this->Email = new TestEmail();
```

Comment:
In the future it would be easier to review if new features and larger scale refactorings were done separately.